docs/sources/next/javascript-api/k6-browser/page/doubledollar.md

```diff
@@ -8,16 +8,18 @@ description: 'Browser module: page.$$(selector) method'
 
 {{< admonition type="warning" >}}
 
-Use locator-based [`page.locator(selector)`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-browser/page/locator/) instead.
+When possible, use locator-based [`page.locator(selector)`](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-browser/page/locator/) instead.
+
+However, working with `locator`s might not be possible when trying to select an element from a list or table if it's difficult to find a stable and unique way to identify the element.
```

Comment:
I'm not sure I understand what this sentence means :/ Does it mean that we can't select multiple elements with `locator`?

Reply:
Yeah, correct. There have been times where it's not easy to create a unique selector for a specific element.

An example of such a situation could be a catalogue page on a shopping website. The items on the catalogue will change often, and so it's difficult to create a stable selector. In this case it's easier to:

1. query for all catalogue elements on the page;
2. pick any;
3. do something with that element.

In PW, the locator API has various methods that could be useful in this scenario: [all](https://playwright.dev/docs/api/class-locator#locator-all), [locator](https://playwright.dev/docs/api/class-locator#locator-locator), [filter](https://playwright.dev/docs/api/class-locator#locator-filter), [nth](https://playwright.dev/docs/api/class-locator#locator-nth), [first](https://playwright.dev/docs/api/class-locator#locator-nth) etc. Until we implement these, `$$` still has a place in the k6 browser module.

Reply:
Yep, thanks for the explanation. Actually, I was referring to the sentence. Can we make it clearer and simpler to understand for others? 😄 Maybe like:

```suggestion
However, using `locator`s may not always work when selecting an element from a list or table, especially if there isn't a reliable way to consistently identify a single element (e.g., due to changing or similar attributes). In such cases, $$ remains useful.
```

Reply:
Thanks for the suggestion, i like it 🙂

Reply:
👍 :)